### DISCUSSION TITLE
家用电脑上ubuntu环境使用1panel安装file brower后输入127.0.0.1:40071无法访问！

### DISCUSSION THREAD

**Discussion by hubuguiqu** [1Panel-dev/1Panel#5512] [Q&A]
### 1Panel 版本

v1.10.10-lts

### 问题描述

如标题所述

### 重现步骤

浏览器返回：无法建立到 127.0.0.1:40071 服务器的连接

**Comment by wanghe-fit2cloud** [comment]
1Panel 是采用容器化方案安装的应用。
在安装应用时，需要开启 `端口外部访问` 选项，如图所示：
<img width="572" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/46734380/3933a88a-7bd3-4866-b6e1-9f9c88887afa">

> 访问应用时要使用服务器 IPv4 地址，而不是 127.0.0.1。
